.storybook/main.js

```diff
@@ -58,7 +59,16 @@ module.exports = {
             loader: 'postcss-loader',
             options: {
               postcssOptions: {
-                plugins: [postcssShopify()],
+                plugins: [
+                  postcssShopify(),
+                  pxtorem({
+                    rootValue: 10,
+                    replace: true,
+                    propList: ['*'],
```

Comment:
Targeting all selectors for testing. We might have to add to the `selectorBlackList` below if we notice anything weird